### DISCUSSION TITLE
Why are breaking changing being introduced in a patch release?

### DISCUSSION THREAD

**Discussion by AnandChowdhary** [prisma/prisma#12589] [Q&A] [answered]
In [v3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1), which is a patch release, there were breaking changes to how Prisma works with MongoDB. I thought Prisma adheres to semver and would have expected a major version bump, or at least a minor bump; in general, patch releases are known to be backwards compatible so this is concerning.

**Comment by austincrim** [comment, marked as answer]
Hey @AnandChowdhary 👋 ,

MongoDB support is still technically a **preview feature** and as such does not fully respect semver yet. Once it goes to **general availability**, it will follow all semver rules like the rest of our stable API surface.

You can read more about the difference in maturity levels [in our docs](https://www.prisma.io/docs/about/prisma/releases).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
